Client/Application/ClientPreferences.swift

```diff
@@ -78,5 +78,13 @@ extension Preferences {
         /// The toggles states for clear private data screen
         static let clearPrivateDataToggles = Option<[Bool]>(key: "privacy.clear-data-toggles", default: [])
     }
+    final class Review {
```

Comment:
If we move `requestReviewIfNecessary` to `BraveShared`, this needs to be moved to `BraveShared/Preferences.swift`